### ISSUE TITLE
server returns 201 for friendbot when it should return 202

### ISSUE THREAD

**Issue by thejollyrogers** [stellar/go-horizon#26]
from the HTTP protocol specification:

"The origin server MUST create the resource before returning the 201 status code. If the action cannot be carried out immediately, the server SHOULD respond with 202 (Accepted) response instead."

Since the account is not created until the transaction is applied, and a call to /accounts/<id> with the address will fail with 404 not found immediately after, the server should respond with a 202 status for friendbot.

**Comment by nullstyle** [comment]
That's making some big assumptions about what resource the /friendbot action is creating. It's just a simple utility function that is useful but not really something that I want to build a stable protocol around.  

IMO, making this change is pedantic and doesn't really buy anyone much.  I'm not opposed to a PR and will include it, but I personally don't care about making this change.

**Comment by thejollyrogers** [comment]
I think it's pretty clear what resource the /friendbot action is creating: it's creating the account. Also, I disagree this change is pedantic or perfectionist - the API was created using HTTP status code 201, which is pretty darn specific. I noticed that even though it was returning 201, the resource wasn't actually created, and never created if stellar-core had lost consensus. Figured since you had returned such a specific status code in the first place, you might care to make it accurate. Obviously I was mistaken, no worries.
